src/components/MetadataSources/SourceManagement/Contact/contact-util.test.js

```diff
@@ -28,7 +25,7 @@ describe('contact-util - handleContactSelected', () => {
         { externalId: 'old-id', name: 'old-name', type: 'old-type' },
         { externalId: '', name: '', type: '' },
       ],
-      update: jest.fn(),
+      update: doUpdate
```

Comment:
As `fields` variable is not changing you can assign it as const.

Reply:
I need `update: doUpdate` here to check if it has been called with the right values:
`expect(doUpdate).toHaveBeenCalledWith(0, { ... }`

Reply:
Sorry, I was talking about the `fields` variable.
```
let fields;
```
You can define it as `const`. There is no need for reinitializing this variable with the same values for each test.